src/utils/transactionUtils/transactionUtils.ts

```diff
@@ -229,3 +255,13 @@ export function getArNSMappingByInteractionType(
   }
   return mapping;
 }
+
+export function getInteractionTypeFromField(field: string) {
+  switch (field) {
+    case 'name':
+      return ANT_INTERACTION_TYPES.SET_NAME;
+
+    default:
+      return undefined;
+  }
+}
```

Comment:
consider using an object for these mappings. 

e.g.
```
export const FieldToInteractionMap = {
    'name': ANT_INTERACTION_TYPES.NAME
}
```
and you can access the object directly or update the function
```
export function  getInteractionTypeFromField(field: string): {
    // maybe check it exists and throw error if it doesn't
    return FieldToInteractionMap[field]
}
```